lib/rialto/etl/cli/publications.rb

```diff
@@ -1,27 +1,114 @@
 # frozen_string_literal: true
 
-require 'csv'
 require 'fileutils'
-require 'rialto/etl/cli/composite_etl'
+require 'jsonpath'
 
 module Rialto
   module Etl
     module CLI
-      # Publications iterates over the people in the input file CSV and for each
-      # calls extract, transform, and load.
-      class Publications < CompositeEtl
-        protected
+      # Publications grabs all Stanford University-affiliated publications from
+      # the WoS API, then transforms and loads them.
+      class Publications < Thor
+        option :input_directory,
+               default: 'data',
+               banner: 'INPUT',
+               desc: 'Name of directory with data to be transformed',
+               aliases: '-d'
+        option :output_directory,
+               default: 'data',
+               banner: 'OUTPUT',
+               desc: 'Name of directory to write transformed data to',
+               aliases: '-o'
+        option :force,
+               default: false,
+               type: :boolean,
+               banner: 'FORCE',
+               desc: 'Overwrite files that already exist',
+               aliases: '-f'
+        option :skip_extract,
+               default: false,
+               type: :boolean,
+               banner: 'SKIP_EXTRACT',
+               desc: 'Skip extract step'
+        option :skip_load,
+               default: false,
+               type: :boolean,
+               banner: 'SKIP_LOAD',
+               desc: 'Skip load step'
 
-        def file_prefix
-          'wos'
+        desc 'load', 'Extract, load, and transform for all Stanford University publications in WoS'
```

Comment:
What happened to only retrieving for a specified time period, e.g., last 30 days?

Reply:
@justinlittman That is ticketed for later. The changes in this PR should make that work easier to pull off. (I've already started working on it a wee bit in a branch off this branch, in fact.)